api/lanes.py

```diff
@@ -132,7 +131,9 @@ def create_default_lanes(_db, library):
         estimates = library.estimated_holdings_by_language()
         large, small, tiny = _lane_configuration_from_collection_sizes(estimates)
     priority = 0
-    for language in large:
+
+    if large and len(large) > 0:
+        language = large[0]
```

Comment:
This seems like a fairly large change, since we were explicitly trying to create lanes for each `large` language before. I think we should at least add tests for the two branches here:

- a configuration with multiple large languages
- a collection with no configuration, that gets this language from `estimated_holdings_by_language`

Reply:
Perhaps this is not  the right solution to the problem.  Is there any case where we would want more than one set of default lanes?

Reply:
@dbernstein I think this is a good solution to the problem, since the duplicate lanes are causing so many issues. I would like to see tests added for the cases I mentioned though.